plugins/orbslam2_plugin/orbslam2_sensors.py

```diff
@@ -0,0 +1,105 @@
+import os
+import math
+import yaml
+from typing import Any, Optional
+from tempfile import mkstemp
+
+import gym
+import numpy as np
+
+import orbslam2
+from core.base_abstractions.task import SubTaskType, Task
+from core.base_abstractions.sensor import Sensor, RGBSensor, DepthSensor
+from core.base_abstractions.misc import EnvType
+from plugins.robothor_plugin.robothor_environment import RoboThorEnvironment
+from plugins.robothor_plugin.robothor_sensors import RGBSensorRoboThor, DepthSensorRoboThor, GPSCompassSensorRoboThor
+from plugins.robothor_plugin.robothor_tasks import PointNavTask
+from plugins.orbslam2_plugin.orbslam2_util import state_to_pose, matrix_to_euler_angles
+from utils.misc_utils import prepare_locals_for_super
+
+
+# check Camera.fps, Camera.bf, ThDepth, DepthMapFactor
+# parameterize ORBextractor
+def init_settings(w, h, fov):
+    f_x = w / (2 * math.tan(math.radians(fov / 2)))
+    f_y = h / (2 * math.tan(math.radians(fov / 2)))
+    c_x, c_y = w / 2, h / 2
+    return {
+        'Camera.fx': f_x, 'Camera.fy': f_y, 'Camera.cx': c_x, 'Camera.cy': c_y,
+        'Camera.k1': 0.0, 'Camera.k2': 0.0, 'Camera.p1': 0.0, 'Camera.p2': 0.0,
+        'Camera.width': w, 'Camera.height': h, 'Camera.fps': 0.0, 'Camera.bf': 40.0, 'Camera.RGB': 1,
+        'ThDepth': 200.0, 'DepthMapFactor': 1.0,
+        'ORBextractor.nFeatures': 1000, 'ORBextractor.scaleFactor': 1.2, 'ORBextractor.nLevels': 8,
+        'ORBextractor.iniThFAST': 20, 'ORBextractor.minThFAST': 1,
+        'Viewer.KeyFrameSize': 0.05, 'Viewer.KeyFrameLineWidth': 1, 'Viewer.GraphLineWidth': 0.9,
+        'Viewer.PointSize':2, 'Viewer.CameraSize': 0.08, 'Viewer.CameraLineWidth': 3,
+        'Viewer.ViewpointX': 0, 'Viewer.ViewpointY': -0.7, 'Viewer.ViewpointZ': -1.8, 'Viewer.ViewpointF': 500
```

Comment:
Would be great to have a comment describing where we should go to find documentation about these parameters.